openpdf/src/main/java/com/lowagie/text/pdf/hyphenation/SimplePatternParser.java

```diff
@@ -101,6 +102,11 @@ public void parse(InputStream stream, PatternConsumer consumer) {
             SimpleXMLParser.parse(this, stream);
         } catch (IOException e) {
             throw new ExceptionConverter(e);
+        } finally {
```

Comment:
Not sure if this is needed. closing a Stream in a Method which didn't open it, is not really a good think.